src/lib/server/phoenixd.ts

```diff
@@ -34,6 +34,17 @@ export async function phoenixdBalance(): Promise<{ balanceSat: number; feeCredit
 	return await res.json();
 }
 
+export async function phoenixdGetBolt12(): Promise<string> {
+	const res = await fetch(`${runtimeConfig.phoenixd.url}/getoffer`, {
+		headers: {
+			Authorization: `Basic ${Buffer.from(`:${runtimeConfig.phoenixd.password}`).toString(
+				'base64'
+			)}`
+		}
+	});
+
+	return await res.text();
```

Comment:
It should check `res.ok` first, and throw an error if not.

Also, https://phoenix.acinq.co/server/api#get-bolt12-offer seems to say it's always the same address, interesting.

Can you check if the API call still works with a fresh node (without any data associated)?

If it doesn't work, what does an error look like? Sometimes the phoenixd server doesn't probably send an HTTP error code (If it works, then we can merge the PR as is, after the check on `res.ok`)